app/models/knight.rb

```diff
@@ -1,2 +1,17 @@
 class Knight < ChessPiece
+  XY_MOVE_FACTOR = 1
+  ALT_XY_MOVE_FACTOR = 2
```

Comment:
is there a more descriptive name for these constants?

I'm not sure I really know what they mean.